profiles/common/modules/custom/ecas/includes/ecas.constants.inc

```diff
@@ -49,3 +49,28 @@ define('ECAS_GATEWAY', 2);
 define('ECAS_DEFAULT_PAGES', '');
 define('ECAS_LOGIN_EXLUDE_PAGES', 0);
 define('ECAS_LOGIN_INCLUDE_PAGES', 1);
+
+/**
+ * ECAS group for CEM team users.
+ */
+define('ECAS_CEM_ECAS_GROUP', 'COMM_CEM');
```

Comment:
Why this here now?!

Reply:
I grouped the constants that I have under the hand there. Apparently the "ecas.constants.inc" is designed for this.